CHANGES/7.feature

```diff
@@ -0,0 +1 @@
+Return execution_environment to importer result
```

Comment:
Recommend this say something like "Move execution_environment from docs_blob to top-level importer result" to be clear to those looking at changelog